content/en/data_jobs/databricks.md

```diff
@@ -184,6 +184,16 @@ In Datadog, view the [Data Jobs Monitoring][6] page to see a list of all your Da
    }
    {{< /highlight >}}
 
+### Set up Data Jobs Monitoring with Databricks Networking Restrictions
+With the default [Databricks Networking Restrictions][12], Datadog may not have access to your Databricks APIs, which can prevent three key requirements for Data Jobs Monitoring from functioning properly:
+1. Ensure that the agent can be downloaded onto the Databricks cluster using the curl command in the init script.
+2. Verify that the Datadog crawler has sufficient permission to access the Databricks workspace.
+3. Confirm that traces and metrics can be sent.
+
+Allow-listing these specific [IP addresses][13] allows your cluster to perform all these interactions with Datadog services. Please see [Databricks documentation][14] for more details on how to manage IP access lists in Databricks.
```

Comment:
```suggestion
If you are controlling Databricks API access through IP access lists, allow-listing Datadog's specific [IP addresses][13] allows your cluster to perform all these interactions with Datadog services. Please see [Databricks documentation][14] for more details on how to manage IP access lists in Databricks.
```

Reply:
Note: The IP access list is actually going to be different depending on the customer site (DC), so we should see if we can make that configurable in the docs or link to the different ones based on Datadog site https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site

Reply:
Done with help from documentation folks !